### ISSUE TITLE
[Accessibility] - NVDA and Talk Back read out statements twice in web dropin component

### ISSUE THREAD

**Issue by OlgaBogachuk** [Adyen/adyen-web#1334]
**Describe the bug**
Some statements of screen readers are repeated twice. Reproduced with NVDA on Win 10 and Talk Back on Android.

**To Reproduce**
Steps to reproduce the behavior under Win 10 with NVDA:

1.	Run NVDA
2.	Go to payment and choose Credit Card
3.	Tab to Credit Card number field and input 1 digit -> the digit is read out twice
4.	Tab to the Expiry Date field -> The text ‘Incomplete field’ for Credit Card Number field is repeated twice
5.	Type some digit into Expiry Date field -> the digit is read out twice.
6.	Tab to CVC / CVV field and type some value -> the digit is read out twice

Steps to reproduce the behavior under Android/Chrome with Talk Back:

1.	Run Talk Back:
2.	Go to payment and choose Credit Card
3.	Tab to Credit Card number field and input ‘1’ -> ‘1’ is read out twice
4.	Input the second digit. e.g., ‘2’ -> the speech is following: ‘2’, then ‘12’ then ‘2’
5.	Tab to the Expiry Date field and input ‘0’-> ‘0’ is read out twice 
6.	Input the second digit into Expiry Date field, e.g., ‘3’ -> the speech is following: ‘3’, then ‘3’ then ‘03’
7.	Tab to CVC / CVV field and type ‘7’ -> the digit is read out twice
8.	Input the second digit. e.g., ‘3’ -> the speech is following: ‘3’, then ‘73’ then ‘3’
9.	Tab to name on Card filed and input some letter -> it is repeated twice

**Expected behavior**
The text that user is typing in, and the error messages should be read out once.

**Screenshots**
![2021-10-21_18-40-24](https://user-images.githubusercontent.com/92933025/138311495-3fe1a647-65b1-4ced-be26-37a9891ad519.png)

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser Chrome
 - Version 94.0.4606.81 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
 - Device: Redmi 3S
 - OS: Android 6.0.1 MMB29M
 - Browser Safari
 - Version 94.0.4606.85
